CONTRIBUTING.md

```diff
@@ -7,14 +7,14 @@ Please note we have a code of conduct, please follow it in all your interactions
 
 ## Pull Request Process
 
-1.  Ensure any install or build dependencies are removed before the end of the layer when doing a
-    build.
-2.  Update the README.md with details of changes to the interface, this includes new environment
-    variables, exposed ports, useful file locations and container parameters.
-3.  Increase the version numbers in any examples files and the README.md to the new version that this
-    Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
-4.  You may merge the Pull Request in once you have the sign-off of two other developers, or if you
-    do not have permission to do that, you may request the second reviewer to merge it for you.
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a
+   build.
+1. Update the README.md with details of changes to the interface, this includes new environment
+   variables, exposed ports, useful file locations and container parameters.
+1. Increase the version numbers in any examples files and the README.md to the new version that this
+   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
+1. You may merge the Pull Request in once you have the sign-off of two other developers, or if you
```

Comment:
All the changes in CONTRIBUTING were just markdown lint

Reply:
Dang - must have to do with spacing / newlines. In markdown for numbered lists just repeat 1. and it works, assuming you do it right :-). I was just linting anyway - no good deed unpunished. I'll fix it